packages/prc/prc.0.1.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Utilities for precision-recall curves"
+description: """
+
+This library provides functions to compute precision-recall curves, as
+well as several methods to compute their AUC (area under curve)
+"""
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Philippe Veber"]
+license: "CeCILL-B"
+tags: ["precision" "recall" "statistics" "machine-learning"]
+homepage: "https://github.com/pveber/prc/"
+bug-reports: "https://github.com/pveber/prc/issues"
+depends: [
+  "alcotest" {with-test}
+  "core_kernel" {>= "v0.12.0"}
+  "dune" {>= "2.4"}
+  "gsl"
+  "ppx_jane"
+  "ppx_deriving" {with-test}
+  "sklearn" {with-test}
+]
+depopts: [
+  "ocaml-r" {>= "v0.5.0"}
+]
```

Comment:
```suggestion
depopts: [
  "ocaml-r"
]
conflicts: [
  "ocaml-r" {< "0.5.0"}
]
```
Unless the package has some kind of weird setup conflict should almost always be used. Also there was a v prefix which made the version never attainable.